EntAppProject/src/main/resources/templates/JBUserProfile.html

```diff
@@ -14,7 +14,7 @@
  
 <h1><span>Job</span>Book</h1>
 
-<h2>Welcome, (User Profile)</h2>
+<h2>Welcome, <span th:text="${userDTO}"></span> </h2>

```

Comment:
Added a Model that inserted the username instead of the text placeholder (user Profile)

Reply:
Good... as long as userDTO is always populated.